extension/json/src/include/json_utils.h

```diff
@@ -29,21 +33,33 @@ class JsonMutWrapper {
     explicit JsonMutWrapper(yyjson_mut_doc* ptr) : ptr{ptr} {}
     ~JsonMutWrapper();
     JsonMutWrapper(JsonMutWrapper& other) = delete;
-    JsonMutWrapper(JsonMutWrapper&& other) { std::swap(ptr, other.ptr); }
+    JsonMutWrapper(JsonMutWrapper&& other) {
```

Comment:
use JsonMutWrapper(JsonMutWrapper&& other) = default;

Reply:
Both src and dst pointers in a default move end up sharing the same address, causing a double free.